master/getting-started/kubernetes/installation/app-layer-policy.md

```diff
@@ -0,0 +1,227 @@
+
+- [Installing {{site.prodname}} for policy and networking (recommended)](calico)
+
+- [Installing {{site.prodname}} for policy and flannel for networking](flannel)
+
+- [Installing {{site.prodname}} for policy (advanced)](other)
+
+# Install Istio
+
+Application Layer Policy requires you to use Istio in your cluster to function correctly. We support Istio version 0.6.0 or newer.
+
+Install Istio using the following command
+
+	kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/manifests/istio.yaml
+
+If you would like to install a different version of Istio or inspect the changes we have made to the standard Istio install manifest, see the [Appendix](#modifying-istio-manifests).
+
+# Enable the Istio Sidecar Injector
+
+The Sidecar Injector automatically modifies pods as they are created to work with Istio. It adds the Istio Proxy and {{site.prodname}} components as sidecar containers.
+
+1. Follow the [Automatic Sidecar Injection instructions](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection) to install the Sidecar Injector and enable it in your chosen namespace(s)
+1. Apply the following ConfigMap to enable injection of {{site.prodname}} components alongside the Istio Proxy.
+
+```
+kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/manifests/istio-inject-configmap.yaml
+```
+
+
+If you would like to install a different version of Istio or inspect the changes we have made to the standard Sidecar Injector ConfigMap, see the [Appendix](#modifying-istio-manifests)
+
+# Label Namespaces for Application Layer Policy
+
+Application Layer Policy is only enforced on pods that are started with the Istio Proxy and Dikastes sidecars.  Pods that do not have these sidecars will only enforce standard Calico network policy.
+
+You can control this on a per-namespace basis.  To enable Istio and Application Layer Policy in a namespace, add the label `istio-injection=enabled`.
+
+	kubectl label namespace <your namespace name> istio-injection=enabled
+
+If the namespace already has pods in it, you will have to recreate them for this to take effect.
+
+**Note**: The Istio Proxy must be able to communicate with the `istio-pilot.istio-system` Service. If you apply any egress policies to your pods, you *must* enable access. For example, you could [apply a NetworkPolicy](manifests/allow-istio-pilot.yaml).
+{: .alert .alert-info}
+
+# Appendix
+
+## Modifying Istio Manifests
+
+Instead of installing from our pre-modified Istio manifests, you may wish to customize your Istio install or use a different Istio version.  This section walks you through the necessary changes to a generic Istio install manifest to allow Application Layer Policy to operate.
```

Comment:
Not sure we want to capitalize "application layer policy" I would recommend instead a lowercase treatment (throughout)